docs/type_inference.md

```diff
@@ -67,7 +67,10 @@ a few problems:
 The solution is as follows:
 
 1. **For most ops without regions** (like `PadOp`): Put all the verification
-   code into the shape functions, and discard verifiers totally.
+   code into the shape functions, and discard verifiers totally. This is
+   applicable if return type inference is feasible. If return type inference
+   is not feasible (like `AddOp`) have a custom verifier to hold
```

Comment:
It is better to use examples where shape inference is not possible even with non-quantized types like  `reshape` and `broadcast_in_dims`. AddOp's type inference in-feasibility is seen oonly with quantized type and that might be difficult to follow.

Reply:
ack, thank you

Reply:
May also be worth mentioning that typically inferable ops, like AddOp, may still need additional verifications when quantization is involved.